runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -89,6 +95,55 @@ fn default_vm_context() -> VMContext {
     };
 }
 
+fn read_file(name: &str) -> std::io::Result<Vec<u8>> {
+    let mut file = File::open(name)?;
+
+    let mut data = Vec::new();
+    file.read_to_end(&mut data)?;
+
+    return Ok(data);
+}
+
+fn write_file(name: &str, data: &Vec<u8>) -> std::io::Result<()> {
+    let mut file = File::create(name)?;
+    file.write(data)?;
+
+    return Ok(());
+}
+
+fn save_contract_to_cache(code: &Vec<u8>, file: &str) {
+    println!("save cache to {}", file);
+    let start = Instant::now();
+    let module = wasmer_runtime::compile(code).unwrap();
+    println!("wasmer_runtime::compile {} μs", start.elapsed().as_micros());
+    let start = Instant::now();
+    let artifact = match module.cache() {
+        Ok(artifact) => artifact,
+        Err(err) => panic!("Cannot compile: {:?}", err),
+    };
+    println!("module.cache {} μs", start.elapsed().as_micros());
+    let start = Instant::now();
+    let code = artifact.serialize().unwrap();
+    println!("artifact.serialize {} μs", start.elapsed().as_micros());
+    write_file(file, &code).unwrap();
+}
+
+fn restore_contract_from_cache(file: &str) {
+    println!("restore cache from {}", file);
+    let bytes = read_file(file).unwrap();
+    let start = Instant::now();
+    let artifact = Artifact::deserialize(&bytes).unwrap();
+    println!("Artifact::deserialize {} μs", start.elapsed().as_micros());
+    unsafe {
+        let compiler = compiler_for_backend(Backend::Singlepass).unwrap();
+        let start = Instant::now();
+        match load_cache_with(artifact, compiler.as_ref()) {
+            Ok(_) => println!("load_cache_with {} μs", start.elapsed().as_micros()),
```

Comment:
```suggestion
            Ok(_) => println!("load_cache_with {:?}", start.elapsed())),
```

duration type has a nice Debug impl which picks the appropriate time unint automatically. `{:.2?}` can be used to not show overly precise fractional parts.